src/core/utils/TreeUtilities.h

```diff
@@ -62,7 +62,7 @@ namespace RevBayesCore {
         void                    rescaleTree(TopologyNode& n, double factor);                                                            //!< rescale tree ages below a node by a factor
         void                    setAges(TopologyNode& n, const std::vector<double>& ages);                                              //!< set ages of a node and children from a vector
         void                    setAgesRecursively(TopologyNode& n, double age);                                                        //!< set age of a node and rescale its children
-        Tree*                   startingTreeInitializer(Tree& treeToChange, std::vector<Taxon>& taxaToCopy);                            //!< make sure starting tree satisfies age constraints, and assign min/max ages to its tips
+        Tree*                   startingTreeInitializer(Tree& treeToChange, std::vector<Taxon>& taxaToCopy, const long& agePrecision);  //!< make sure starting tree satisfies age constraints, and assign min/max ages to its tips
```

Comment:
doesn't need to be a `const long&` and instead can be `long` because a constant reference takes up as much space as a long.